src/main/java/org/carlmontrobotics/robotcode2023/Constants.java

```diff
@@ -105,6 +107,9 @@ public static final class Drivetrain {
         public static final double[] yPIDController = {4, 0.0, 0.0};
         public static final double[] thetaPIDController = {0.10, 0.0, 0.001};
 
+        //public static final double[] tolerance = {Units.inchesToMeters(1), Units.inchesToMeters(1), 5};
+
+
```

Comment:
Why was this added?

Reply:
this was for the original CorrectToPoint command, but now the CorrectToPoint uses positionTolerance and velocityTolerance

Reply:
The remove the line instead of commenting it out.